### ISSUE TITLE
The remove-feature stopped working in v0.1.10(?)

### ISSUE THREAD

**Issue by RonaldTreur** [dciccale/node-htmlprocessor#13]
I'm not sure why, but somewhere in between v0.1.9 and v0.1.10, the remove-functionality stopped working.

Take the following example:
```html
<!-- build:remove:testa -->
 test A
<!-- /build -->
<!-- build:remove:testb -->
 test B
<!-- /build -->
```

When using grunt-processhtml like so:
```js
testa: {
	options: {
		strip: true
	},
	files: [...]
}
```

The result is (surprisingly):
```html
 test A
<!-- build:remove:testa -->
 test B
<!-- /build -->
```

whereas v0.1.9 yielded (correctly):
```html
 test A
```

So, even though `strip` worked, `remove` somehow did not.

**Comment by dciccale** [comment]
if you include a starting new line in the top of the file, it works.

however would be a workaround, will try to find a better fix

```html

<!-- build:remove:testa -->
 test A
<!-- /build -->
<!-- build:remove:testb -->
 test B
<!-- /build -->
```

**Comment by dciccale** [comment]
will be fixed in v0.2.0
